src/test/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElementTest.java

```diff
@@ -152,4 +158,52 @@ alphanumeric chars (only ASCII)
 
         assertEquals("dir____ ___/xyz/\\a.b-c", result);
     }
+
+    @Test
+    void pathToFileInfo_should_return_same_path_for_physical_and_normal() throws Exception {
+        var doc = readDocumentFromString(
+            "<file filepath=\"data/path/to/file1.txt\" xmlns=\"http://easy.dans.knaw.nl/schemas/bag/metadata/files/\" xmlns:dcterms=\"http://purl.org/dc/terms/\">\n"
```

Comment:
Single quotes are better readable than escaped double quotes. The escaped line endings hurt less to my eyes but serve no purpose in this context.